docs/bn/week06/06-1.md

```diff
@@ -0,0 +1,186 @@
+
+##ফলস পজিটিভসঃ
+একটি স্থিরচিত্রের টুকরো অংশের মাঝে মুখাবয়বের চেয়ে ভিন্ন কয়েক ধরণের অবজেক্ট প্রতীয়মান হতে পারে। মডেল ট্রেইনিং ধাপে এই  সব ধরণের ভিন্ন অবজেক্ট গুলো নিরীক্ষণের আওতার বাইরে থেকে যেতে পারে ( সব ধরণের বলতে  অমুখাবয়ব সম্পর্কিত চিত্রের অংশ গুলো দ্বারা গঠিত একটি সম্পূর্ণ সেট)। কাজেই টেস্টিং ধাপে এসে মডেল এর আউটপুট ফলস পজিটিভ দ্বারা অনেকাংশে প্রভাবিত হতে পারে। উদাহরণ স্বরূপ বলা যায় যে, যদি নেটওয়ার্কটি এমন চিত্র সমূহ দ্বারা ট্রেইনড না হয় যেখানে হাতের উপস্থিতি আছে, মডেলটি ত্বকের বর্ণের উপর ভিত্তি করে মুখাবয়ব চিহ্নিত করতে পারে এবং ত্রুটিপূর্ণ শ্রেণীবিভাগের মাধ্যমে চিত্রে হাতের উপস্থিতিকে মুখের সম্যক বলে চিহ্নিত করতে পারে। ভুল চিহ্নিতকরণের মধ্য দিয়ে ফলস পজিটিভস বৃদ্ধি পেতে পারে।
+
+<!--- **Different Face Size:** Not all faces are 30 $\times$ 30 pixels, so faces of differing sizes may not be detected. One way to handle this issue is to generate multi-scale versions of the same image. The original detector will detect faces around 30 $\times$ 30 pixels. If applying a scale on the image of factor $\sqrt 2$, the model will detect faces that were smaller in the original image since what was 30 $\times$ 30 is now 20 $\times$ 20 pixels roughly. To detect bigger faces, we can downsize the image. This process is inexpensive as half of the expense comes from processing the original non-scaled image. The sum of the expenses of all other networks combined is about the same as processing the original non-scaled image. The size of the network is the square of the size of the image on one side, so if you scale down the image by $\sqrt 2$, the network you need to run is smaller by a factor of 2. So the overall cost is $1+1/2+1/4+1/8+1/16…$, which is 2. Performing a multi-scale model only doubles the computational cost.-->
+
+##বিভিন্ন আকারের মুখাবয়বঃ সকল মুখের ছবি ৩০x৩০ পিক্সেলস হতে নাও পারে, সেক্ষেত্রে বিভিন্ন আকারের মুখের ছবি চিহ্নিত হতে না পারে। একটি উপায়ে এই সমস্যাটি উত্তরণ করা যেতে পারে, সেটি হচ্ছে একই চিত্রের বহু মাপের সংস্করণ বের করে। প্রকৃত ডিটেক্টরটি ৩০x৩০ পিক্সেলসের মুখাবয়ব চিহ্নিত করতে পারবে। মডেলটি  আসল চিত্রে অপেক্ষাকৃত ছোট আকারের মুখ চিহ্নিত করতে পারবে যদি আনুপাতিক \sqrt 2 গুণক পরিমাণ আসল চিত্রে পরিবর্তন প্রয়োগ করা হয়, যাতে করে ৩০x৩০ পিক্সেলস প্রয়োগের পরবর্তীতে আনুমানিক ২০x20 পিক্সেলসে পরিবর্তিত হয়ে যায়। বড় আকারের মুখ চিহ্নিত করতে আমরা চিত্রের আকার ছোট করে ফেলতে পারি। এই পদ্ধতিটি প্রয়োগ সহজ, যেহেতু প্রয়োগের অর্ধেক অ্যালগোরিদমিক ব্যয় অপরিবর্তিত আসল চিত্রের প্রস্তুতিকরণে পিছে যায়। বাকি সব নেটওয়ার্কের সম্মিলিত প্রয়োগ খরচ অপরিবর্তিত আসল চিত্রের প্রক্রিয়ার খরচের সমান। নেটওয়ার্কের আকার আসল চিত্রের এক পাশের আকারের চতুর্গুন পরিমাণ, সেহেতু একটি চিত্রকে যদি \sqrt 2 গুণক পরিমাণ ছোট করা হয়, প্রয়োজনীয় নেটওয়ার্কের আয়তন দাঁড়াবে আসল নেটওয়ার্কের আকারের তুলনায় দ্বিগুণ ছোট। তাহলে সর্বমোট খরচ দাঁড়ায়  ১+১/২+১/৪+১/৮+১/১৬... , ২। একটি বহু সংস্করণের মডেল প্রয়োগ করলে কম্পিউটেশনাল খরচ কেবল দ্বিগুণ হবে।
```

Comment:
What happened with the LaTeX here???

Reply:
I had mistakenly converted the LaTex into Bangla too, modified it back to English.